libraries/freertos_plus/standard/crypto/src/iot_crypto.c

```diff
@@ -68,10 +68,10 @@ typedef struct SignatureVerificationState
 /*-----------------------------------------------------------*/
 
 /**
- * @brief Implements libc calloc semantics using the FreeRTOS heap
+ * @brief Implementation of calloc for use by the mbedTLS heap management API
  */
-static void * prvCalloc( size_t xNmemb,
-                         size_t xSize )
+void * pvMbedTLSCalloc( size_t xNmemb,
```

Comment:
For what's it's worth, I have no idea what the variable name "xNmemb" is supposed to represent.